dring/src/main/AndroidManifest.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="org.couchsource.dring.application"
-    android:installLocation="internalOnly">
+    android:installLocation="auto">
```

Comment:
installLocation="internalOnly" is by design. The app relies on ACTION_BOOT_COMPLETED broadcast after restart. External storage app may not receive the broadcast. For more: http://developer.android.com/guide/topics/data/install-location.html#ShouldNot